web/src/main/java/org/jboss/metadata/parser/jbossweb/Attribute.java

```diff
@@ -37,6 +37,8 @@
     // jboss-web attributes in alpha order
     CONFIG_PARSER_CLASS("configParserClass"),
 
+    FLUSH_ON_SESSION_INVALIDATION("flushOnSessionInvalidation"),
```

Comment:
I see this is following attribute name "conventions" in this java class but this is really awful. Everywhere else we use dash-delimited-literals as opposed to camelcasing, I don't see why this would be different for attribute names.

It looks like all the other ones are deprecated by now, so I think we can be consitent with rest of the naming we use in current versions.

Just my 5 cents.